frontend/src/pages/Footer/_Footer.scss

```diff
@@ -0,0 +1,107 @@
+@use "../../components/Basics/" as *;
+
+.footer-nav {
+  background-color: $color-blue-darker;
+  height: 128px;
+  width: 100%;
+  bottom: 0;
+  align-items: center;
+  @include breakpoint-media-max("laptop") {
```

Comment:
Laptop is not a good transition point to mobile, since that includes all screens other than desktop. If you need room to shrink, it's smarter to take it from the 170+ px left and right margins, since those are easily adjustable and hardly noticeable by users. Look at the header for an example. GitHub's footer as well only uses maybe at most 40px, depending on screen size.

Reply:
Included  breakpoint-media-max('smtablet') as transition point to mobile.